### ISSUE TITLE
Simplify external documentation

### ISSUE THREAD

**Issue by jlvandenhout** [iotaledger/iota-wiki#778]
TL;DR: Let's maintain dependencies in the Wiki root, move the CLI to the Wiki repo and use `npx` or `yarn dlx` to run a local development server in external documentation.

We have the need to add the same external documentation to multiple environments, for example, environment-agnostic documentation and library documentation that has versions compatible with multiple environments. We also wanted external documentation to be able to provide its own dependencies, so we configured them to be their own NodeJS packages which we included in our build using Yarn workspaces.

Yarn, however, doesn't allow duplicate workspaces in a worktree. Also, external documentation does not seem to need any other functionality than we provide out of the box already, or could easily add as root dependency when necessary, removing the need for external documentation to be their own packages.

TODO:
- [x] Remove workspaces and install any missing dependencies in the Wiki root package.
- [x] Move the CLI to the Wiki repo and let it use the Next environment configuration.
- [x] Publish the CLI to NPM.
~Remove `package.json` and add `npx` command to README in the external documentation repos:~
- [ ] Update `package.json` to have `"@iota-wiki/cli": "latest"` in `devDependency` and remove `yarn.lock`:
  - [ ] [bee](https:/hub.com/iotaledger/bee)
  - [ ] [blueprints](https:/hub.com/iotaledger/blueprints)
  - [ ] [chronicle.rs](https:/hub.com/iotaledger/chronicle.rs)
  - [ ] [cli-wallet](https:/hub.com/iotaledger/cli-wallet)
  - [ ] [goshimmer](https:/hub.com/iotaledger/goshimmer)
  - [ ] [hornet](https:/hub.com/iotaledger/hornet)
  - [ ] [identity.rs](https:/hub.com/iotaledger/identity.rs)
  - [ ] [integration-services](https:/hub.com/iotaledger/integration-services)
  - [ ] [introduction-docs](https:/hub.com/iotaledger/introduction-docs)
  - [ ] [inx-chronicle](https:/hub.com/iotaledger/inx-chronicle)
  - [ ] [inx-coordinator](https:/hub.com/iotaledger/inx-coordinator)
  - [ ] [inx-dashboard](https:/hub.com/iotaledger/inx-dashboard)
  - [ ] [inx-faucet](https:/hub.com/iotaledger/inx-faucet)
  - [ ] [inx-indexer](https:/hub.com/iotaledger/inx-indexer)
  - [ ] [inx-mqtt](https:/hub.com/iotaledger/inx-mqtt)
  - [ ] [inx-participation](https:/hub.com/iotaledger/inx-participation)
  - [ ] [inx-poi](https:/hub.com/iotaledger/inx-poi)
  - [ ] [inx-spammer](https:/hub.com/iotaledger/inx-spammer)
  - [ ] [IOTA-2.0-Research-Specifications](https:/hub.com/iotaledger/IOTA-2.0-Research-Specifications)
  - [ ] [iota.js](https:/hub.com/iotaledger/iota.js)
  - [ ] [iota.rs](https:/hub.com/iotaledger/iota.rs)
  - [ ] [streams](https:/hub.com/iotaledger/streams)
  - [ ] [stronghold.rs](https:/hub.com/iotaledger/stronghold.rs)
  - [ ] [wallet.rs](https:/hub.com/iotaledger/wallet.rs)
  - [ ] [wasp](https:/hub.com/iotaledger/wasp)
  - [ ] [Zebra-Iota-Edge-SDK](https:/hub.com/ZebraDevs/Zebra-Iota-Edge-SDK)

**Comment by jlvandenhout** [comment]
For reference: `npx` or `yarn dlx` won't work, because we need Docusaurus to be installed locally, so a `package.json` is necessary anyway.

**Comment by jlvandenhout** [comment]
@Dr-Electron I guess this can be closed? We only have the TIPs repo as submodule, but I think we should handle any changes to that approach in separate issues.

**Comment by Dr-Electron** [comment]
Yes
